doc/source/management/ssl-improvement.rst

```diff
@@ -0,0 +1,42 @@
+.. _percona-server.management.ssl:
+
+================================================================================
+SSL Improvements
```

Comment:
Points 6 and Points 7 not addressed (discussion on Slack)

Reply:
ready for review